### PR TITLE
Add citadel replicaCount flag back and workloadCertTtl

### PR DESCRIPTION
**Pull request by wattli**
To support multi-citadel and short cert TTL for perf test

### CODE REVIEW

**Comment by myidpt** [comment]
Did we fully verify that multiple Citadels will write the workload secrets only once?

**Comment by myidpt** [comment]
Chatted offline with Tao.
/lgtm
/approve

File: install/kubernetes/helm/istio/charts/security/templates/deployment.yaml [hunk 1]
```diff
@@ -60,6 +60,9 @@ spec:
           {{- if .Values.global.trustDomain }}
             - --trust-domain={{ .Values.global.trustDomain }}
           {{- end }}
+          {{- if .Values.workloadCertTtl }}
+            - --workload-cert-ttl={{ .Values.workloadCertTtl }}
```
Comment (myidpt) [inline install/kubernetes/helm/istio/charts/security/templates/deployment.yaml]: Add workloadCertTtl to values.yaml?

Comment (wattli) [reply install/kubernetes/helm/istio/charts/security/templates/deployment.yaml]: done.

**Pull request merged**